File: examples/src/Presentations/Properties/CheckPasswordExample.cs
using System
using System.Collections.Generic
using System.IO
using System.Linq
using System.text
using System.Threading.Tasks
import aspose.slides as slides
using Aspose.slides.Examples.CSharp
using Aspose.slides.Export
using Aspose.slides.Util

namespace CSharp.Presentations.properties
{
    public class CheckPasswordExample
    {
        # The example below demonstrates how to check a password to open a presentation

        public static void Run()
        {
            #Path for source presentation
            pptFile = Path.Combine(RunExamples.GetDataDir_PresentationProperties(), "open_pass1.ppt")

            # Check the Password via IPresentationInfo Interface
            IPresentationInfo presentationInfo = PresentationFactory.Instance.GetPresentationInfo(pptFile)
            bool isPasswordCorrect = presentationInfo.CheckPassword("my_password")
            print("The password \"my_password\" for the presentation is " + isPasswordCorrect)
            
            isPasswordCorrect = presentationInfo.CheckPassword("pass1")
            print("The password \"pass1\" for the presentation is " + isPasswordCorrect)
        }
    }
}

File: examples/src/Presentations/Properties/CheckPasswordExample.py
import aspose.slides as slides

def props_check_password():
    # The example below demonstrates how to check a password to open a presentation
    #Path for source presentation
    dataDir = "./examples/data/"
    outDir = "./examples/out/"

    # Check the Password via IPresentationInfo Interface
    presentationInfo = slides.PresentationFactory.instance.get_presentation_info(dataDir + "props_ppt_with_password.ppt")
    isPasswordCorrect = presentationInfo.check_password("my_password")
    print("The password \"my_password\" for the presentation is " + str(isPasswordCorrect))
    
    isPasswordCorrect = presentationInfo.check_password("pass1")
    print("The password \"pass1\" for the presentation is " + str(isPasswordCorrect))
